pkgs/by-name/ti/tiramisu/package.nix

```
@@ -7,16 +7,14 @@
   vala,
 }:
 
-stdenv.mkDerivation rec {
-  pname = "tiramisu";
-  # FIXME: once a newer release in upstream is available
-  version = "2.0-unstable-2023-03-29";
+stdenv.mkDerivation (finalAttrs: {
+  name = "tiramisu";
+  version = "2.0.20240610";
 
   src = fetchFromGitHub {
     owner = "Sweets";
-    repo = "tiramisu";
-    # FIXME: use the current HEAD commit as upstream has no releases since 2021
-    rev = "5dddd83abd695bfa15640047a97a08ff0a8d9f9b";
+    repo = finalAttrs.name;
```

The suggestion by @pbsds wasn't nitpicking, but rather sound (see https://github.com/NixOS/nixpkgs/issues/310373)

```suggestion
    repo = "tiramisu";
```

It's just not useful to make the repo overridable.